WcaOnRails/config/schedule.yml

```diff
@@ -1,5 +1,5 @@
 clear_stripe:
-  class: "ClearConnectedStripeAccount"
+  class: "ClearConnectedPaymentIntegrations"
```

Comment:
@gregorbg I believe the change requested for `schedule.yml` is already in place - or am I missing something somewhere else in the code?

Reply:
Oh, nevermind :see_no_evil: